account_credit_control/__manifest__.py

```diff
@@ -15,6 +15,7 @@
  'depends': [
      'base',
      'account',
+     'contacts',
```

Comment:
The dependency on `contacts` seems to be necessary only for the configuration menu. To avoid adding it, it might be better to add the menu next to the Credit Control Policies menu.